.github/workflows/diff-change-to-dist.yaml

```
@@ -52,19 +52,30 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const { readFile } = require('node:fs/promises')
-            const { comment } = await import('${{ github.workspace }}/.github/workflows/scripts/comments.mjs')
-
-            // Read diff from previous step
-            const diffText = await readFile('${{ github.workspace }}/dist.diff', 'utf8')
+            const { commentDiff } = await import('${{ github.workspace }}/.github/workflows/scripts/comments.mjs')
 
             // PR information
-            const number = '${{ github.event.pull_request.number }}'
+            const issueNumber = '${{ github.event.pull_request.number }}'
             const commit = '${{ github.event.pull_request.head.sha }}'
 
-            // Add or update comment on PR
-            await comment({ github, context }, number, 'Diff release', '## Changes to dist\n' +
-              '```diff\n' +
-              `${diffText}\n` +
-              '```\n\n' +
-              `SHA: ${commit}\n`
-            )
+            const diffs = [
+              {
+                path: '${{github.workspace}}/dist-js.diff',
+                title: 'JavaScript changes to `dist`',
+                markerText: 'dist-diff-js'
+              },
+              {
+                path: '${{github.workspace}}/dist-css.diff',
+                title: 'Stylesheets changes to `dist`',
+                markerText: 'dist-diff-css'
+              },
+              {
+                path: '${{github.workspace}}/dist-other.diff',
+                title: 'Other changes to `dist`',
+                markerText: 'dist-diff-other'
+              }
+            ]
+
+            for (const diff of diffs) {
+              await commentDiff({github, context, commit}, issueNumber, diff)
```

Mind just running all this code through ESLint again? Just in case

```suggestion
              await commentDiff({ github, context, commit }, issueNumber, diff)
```